.github/workflows/test.yml

```diff
@@ -67,6 +67,19 @@ jobs:
               run: |
                 composer require "laravel/laravel:${{ matrix.laravel }}" "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update
 
+            - name: Get Composer Cache Directory
+              id: composer-cache
+              run: |
+                echo "::set-output name=dir::$(composer config cache-files-dir)"
+              
+            - name: Cache Composer Downloads
+              uses: actions/cache@v2
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: ${{ runner.os }}-composer--${{ hashFiles('**/composer.json') }}
+                  restore-keys: |
+                    ${{ runner.os }}-composer-
```

Comment:
Either this needs to be cut off or it would need to include the laravel version? 🤔

Reply:
Took it from an example. Turns out `restore-keys` are optional and it's used as an alternative key, so I'll just remove it.

Reply:
Removed it and then put it back in 😄 I think we can leave it like this as a default cache key that will always get a hit.
Reference: https://docs.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows#example-using-the-cache-action